tests/fuzz/fuzz_dtlsclient.c

```diff
@@ -0,0 +1,185 @@
+#include <string.h>
```

Comment:
Can you please explain how you generated the corpus files?

I'm not familiar with oss-fuzz tools. When would the corpus files need to be re-generated? When the clients change? When some supporting library changes? Is the format documented somewhere?

Reply:
> Can you please explain how you generated the corpus files?

As far as I can remember, this was some tacky work by hands, with debug prints used with simple client and server...
This certainly can be improved...

> When would the corpus files need to be re-generated?

Oss-fuzz handles that part.
But it is always better to give fuzzing a good start with some corpus.

> Is the format documented somewhere?

Not yet. It is the tcp flow (or udp in case of dlls) from client or from server + some options in the first bytes